index.bs

```diff
@@ -891,15 +850,107 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
+          the MIME type of |contentType| is [=supported by the user agent=], or
+          <code>failure</code> if |contentType| failed to be parsed:
+        </p>
+        <ol>
+          <li>
+            Let |mimeType| be the result of running <a>parse a
+            MIME type</a> with |contentType|.
+          </li>
+          <li>
+            If the value of |mimeType| is <code>failure</code>,
+            return <code>failure</code>.
+          </li>
+          <li>
+            If |encodingOrDecodingType| is {{MediaEncodingType/webrtc}} ({{MediaEncodingType}})
+            or {{MediaDecodingType/webrtc}} ({{MediaDecodingType}}):
+            <ol>
+              <li>
+                <p>
+                  If |mimeType| is not the list of RTP Payload Format Media Types in
+                  [[IANA-RTP-PARAMETERS]], return <code>false</code>.
+                </p>
+                <p class="note">
+                  The codec name is typically specified as subtype and zero or more
+                  parameters may be present depending on the codec.
+                </p>
+              </li>
+              <li>
+                If |mimeType| is not [=supported by the user agent=], return
+                <code>false</code>.
+              </li>
+              <li>
+                Return <code>true</code>.
+              </li>
+            </ol>
+          </li>
+          <li>
+            If |media| is <code>audio</code> and |mimeType|'s [=MIME type/type=]
+            is neither <code>audio</code> nor <code>application</code>,
+            return <code>false</code>.
+          </li>
+          <li>
+            If |media| is <code>video</code> and |mimeType|'s [=MIME type/type=]
+            is neither <code>video</code> nor <code>application</code>,
+            return <code>false</code>.
+          </li>
+          <li>
+            If the combined <code>type</code> and <code>subtype</code> members
+            of |mimeType| imply a single media codec:
+            <ol>
+              <li>
+                If the <code>parameters</code> member of |mimeType|
+                is not empty, return <code>failure</code>.
+              </li>
+              <li>
+                If |mimeType| is not [=supported by the user agent=], return
+                <code>false</code>.
+              </li>
+              <li>
+                Return <code>true</code>.
+              </li>
+            </ol>
+          </li>
+          <li>
+            Otherwise:
+            <ol>
+              <li>
+                If the <code>parameters</code> member of |mimeType|
+                does not contain a single key named "codecs", return
+                <code>false</code>.
+              </li>
+              <li>
+                If the value of <code>mimeType.parameters["codecs"]</code>
+                does not describe a single media codec, return
+                <code>false</code>.
+              </li>
```

Comment:
Why does it matter if a single media codec is listed? The restriction does not seem necessary given that the next bullet point gives freedom to the user agent to reject the mime type if it does not support it.
Note that there are conversations in MPEG about describing a single, layered video stream stored in a single track using multiple values in the `codecs` parameter.

Reply:
It is my understanding that MediaCapabilities is about a 1:1 query. The typical use of listing multiple codecs is for doing audio&video, and test audio&video separately.

Reply:
I've filed https://github.com/w3c/media-capabilities/issues/235 to follow this up.